persistence-api/src/main/java/io/stargate/db/query/builder/QueryBuilderImpl.java

```diff
@@ -346,6 +357,51 @@ public void writeTimeColumn(String columnName, String alias) {
     this.writeTimeColumnAlias = alias;
   }
 
+  public void count(String countColumnName) {
+    count(countColumnName, null);
+  }
+
+  public void count(String countColumnName, String alias) {
```

Comment:
If we're introducing selector aliases, why not support them for ordinary columns too?

Reply:
Again - I am doing the same as for writeTimestamp. I don't need an alias for other columns. If we have a demand for this feature, we can address that in the separate PR.

Reply:
added a comment about alias to the thread on line 91... resolving this conversation.